website/pages/docs/builders/amazon/chroot.mdx

```diff
@@ -289,6 +289,7 @@ variables are available:
   building the AMI.
 - `SourceAMI` - The source AMI ID (for example `ami-a2412fcd`) used to build
   the AMI.
+- `SourceAMICreationDate` - The source AMI creation date
```

Comment:
Can you please add an example of date here for better docs ?

Reply:
Added an example. Thanks for the comment.